### PR TITLE
Raise ES output plugin request timeout to 600s

### PR DESCRIPTION
**Pull request by richm**
this is the backport to release-1.5

### CODE REVIEW

**Comment by richm** [comment]
no automated test/merge for this branch

File: fluentd/configs.d/openshift/es-copy-config.conf [hunk 1]
```diff
@@ -17,11 +17,16 @@
...
+      flush_interval "#{ENV['ES_FLUSH_INTERVAL'] || '1s'}"
       max_retry_wait "#{ENV['ES_RETRY_WAIT'] || '300'}"
       disable_retry_limit true
       buffer_type file
       buffer_path '/var/lib/fluentd/buffer-es-copy-config'
-      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
-      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '1m' }"
+      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '32' }"
+      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '8m' }"
+      # the systemd journald 0.0.8 input plugin will just throw away records if the buffer
```
Comment (portante) [inline fluentd/configs.d/openshift/es-copy-config.conf]: I think we can remove this comment entirely.

File: fluentd/configs.d/openshift/es-ops-copy-config.conf [hunk 2]
```diff
@@ -17,11 +17,16 @@
...
+      flush_interval "#{ENV['OPS_FLUSH_INTERVAL'] || ENV['ES_FLUSH_INTERVAL'] || '1s'}"
       max_retry_wait "#{ENV['OPS_RETRY_WAIT'] || ENV['ES_RETRY_WAIT'] || '300'}"
       disable_retry_limit true
       buffer_type file
       buffer_path '/var/lib/fluentd/buffer-es-ops-copy-config'
-      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
-      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '1m' }"
+      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '64' }"
+      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '8m' }"
+      # the systemd journald 0.0.8 input plugin will just throw away records if the buffer
```
Comment (portante) [inline fluentd/configs.d/openshift/es-ops-copy-config.conf]: I think we can remove this comment entirely.

File: fluentd/configs.d/openshift/es-copy-config.conf (hunk 1 above)
Comment (richm) [reply fluentd/configs.d/openshift/es-copy-config.conf]: @portante file a new issue - this PR is a cherry-pick of a cherry-pick of commits that have already been pushed

File: fluentd/configs.d/openshift/es-ops-copy-config.conf (hunk 2 above)
Comment (richm) [reply fluentd/configs.d/openshift/es-ops-copy-config.conf]: @portante file a new issue - this PR is a cherry-pick of a cherry-pick of commits that have already been pushed

Comment (portante) [reply fluentd/configs.d/openshift/es-ops-copy-config.conf]: I see this in master in two places which do not appear to be related to this:
 * https://github.com/openshift/origin-aggregated-logging/blob/b93e416f6749ab668ceb7b01395d7158e984a565/fluentd/configs.d/filter-pre-mux-client.conf#L13
 * https://github.com/openshift/origin-aggregated-logging/blob/b93e416f6749ab668ceb7b01395d7158e984a565/fluentd/configs.d/user/secure-forward.conf#L17

Not sure how a cherry pick could have pulled back those messages to here.

**Comment by jcantrill** [comment]
1.5 backport of #698

**Comment by jcantrill** [comment]
This is a backport of #698

**Pull request merged**